pkg/agent/sysadvisor/types/helper.go

```diff
@@ -67,6 +67,41 @@ func (ci *ContainerInfo) Clone() *ContainerInfo {
 	return clone
 }
 
+func (ci *ContainerInfo) Equal(c *ContainerInfo) bool {
```

Comment:
why not use apiequality.Semantic.DeepEqual?

Reply:
done